neo/SDK/SC/ContractClient.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Neo.Network.P2P.Payloads;
+using Neo.Network.RPC;
+using Neo.Network.RPC.Models;
+using Neo.SDK.TX;
+using Neo.SmartContract;
+using Neo.SmartContract.Manifest;
+using Neo.VM;
+using Neo.Wallets;
+
+namespace Neo.SDK.SC
+{
+    public class ContractClient
```

Comment:
This is a wrapper of `RpcClient`, right?

Reply:
This a class for TestInvode, Invoke or Deploy smart contract.

Reply:
Then it should be moved into the namespace `Neo.Network.RPC`.

Reply:
This class is used for smart contract related operations. It just calls some methods from RpcClient.

Reply:
It's a wrapper of `RpcClient`. It doesn't call `ApplicationEngine` directly.

Reply:
All of C# SDK functions interact with neo blockchain through `RPC`. We have designed different modules for different purpose. It's more important for SDK developers to know `what modules can do` rather than `how they do`.

Reply:
You need to be more precise in your use of the term "SDK". Developers can use the NEO.dll to build their own clients like neo-cli, plugins for neo-cli or apps that interact with neo-cli via RPC. My recommendation is to call this the "RPC SDK" or the "Remote SDK".

Reply:
Agree with @devhawk

Reply:
Then we should move all the classes under SC and TX to RPC folder, right? @erikzhang

Reply:
Yes.

Reply:
OK.